phpcs.xml.dist

```diff
@@ -198,6 +222,18 @@
     ref="SlevomatCodingStandard.ControlStructures.RequireNullCoalesceOperator"
   />
 
+  <rule ref="SlevomatCodingStandard.TypeHints.PropertyTypeHint">
+      <exclude-pattern>web-stories.php</exclude-pattern>
+      <exclude-pattern>includes/compat/*</exclude-pattern>
+      <exclude-pattern>includes/Services.php</exclude-pattern>
```

Comment:
I think this one can now be removed.

We just have to make the property types in `Services.php` nullable:

```php
	/**
	 * Plugin object instance.
	 */
	private static ?Plugin $plugin = null;

	/**
	 * Service container object instance.
	 */
	private static ?ServiceContainer $container = null;
```

Then there aren't any errors.